.jenkinsci/linux-post-step.groovy

```diff
@@ -16,6 +17,20 @@ def linuxPostStep() {
           echo "Build is not SUCCESS! See core dumps at: https://nexus.iroha.tech/repository/artifacts/iroha/coredumps/${dumpsFileName}"
         }
       }
+
+      // handling build time results
+      if (currentBuild.currentResult == "SUCCESS" && !fuzzingEnabled) {
```

Comment:
Add this :
`def fuzzingEnabled = params.fuzzing`
or just use `params.fuzzing`
Keep in mind it can be `null`, in some case
>  // params are always null unless job is started
>  // this is the case for the FIRST build only.	 
 // So just set this to same value as default.	
  // This is a known bug. See https://issues.jenkins-ci.org/browse/JENKINS-41929

Reply:
Ok, will fix this